modules/virt-about-cluster-network-addons-operator.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * virt/virt-architecture.adoc
+
+:_content-type: CONCEPT
+[id="virt-about-cluster-network-addons-operator_{context}"]
```

Comment:
@phoracek  is this a fair description of network components?

Reply:
Looks fine